app/ante/eth.go

```diff
@@ -237,7 +237,7 @@ func (nvd NonceVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, sim
 	}
 
 	seq := acc.GetSequence()
-	if msgEthTx.Data.AccountNonce != seq {
+	if msgEthTx.Data.AccountNonce < seq {
```

Comment:
why the change?

Reply:
the sequence goes out of order as it takes some time for an account nonce to either be updated or reverted. thus, we need this in order to bypass the nonce for these benchmarking tests. Any tips on how to work around this? I hesitate to merge this branch because of this change

Reply:
I see. Then I'd probably update the docker README and suggest to change this line if someone wants to reproduce the benchmarks

Reply:
https://github.com/ChainSafe/ethermint/pull/550